addons/content-index/src/main/java/org/commonjava/indy/content/index/ContentIndexObserver.java

```diff
@@ -272,8 +272,8 @@ private void propagatePathlessStoreEvent( Iterable<ArtifactStore> stores )
                 final StoreKey key = store.getKey();
 
                 Set<String> paths = new HashSet<>();
-                indexManager.removeAllIndexedPathsForStore( key, indexedStorePath ->{paths.add(indexedStorePath.getPath());} );
-                indexManager.removeAllOriginIndexedPathsForStore( key, indexedStorePath ->{paths.add(indexedStorePath.getPath());} );
+                indexManager.removeAllIndexedPathsForStore( key, indexedStorePath ->{paths.add(indexedStorePath.getPathHash());} );
+                indexManager.removeAllOriginIndexedPathsForStore( key, indexedStorePath ->{paths.add(indexedStorePath.getPathHash());} );
 
```

Comment:
Yeah, I need to back it out. With folo tracking in there, it's not saving us anything anyway. However, I thought any interaction with the content index would already have a path for us to work with?